playbooks/roles/ufw/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Ensure UFW allows SSH (22/TCP)
+  ufw:
```

Comment:
Minor quibble, but the convention in the repo is that the module arguments start on the same line:

```
- name: Ensure UFW allows SSH (22/TCP)
  ufw: rule="allow"
       name="OpenSSH"
```